src/plugins/pictureInPicture.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2023 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { React, Tooltip } from "@webpack/common";
+
+export default definePlugin({
+    name: "PitureInPicture",
+    description: "Adds picture in picture to videos (next to the Download button)",
+    authors: [Devs.Lumap],
+    patches: [{
+        find: ".onRemoveAttachment,",
+        replacement: {
+            match: /\.nonMediaAttachment.{0,10}children:\[(.),/gm,
```

Comment:
unless you deliberately mean to replace multiple places
```suggestion
            match: /\.nonMediaAttachment.{0,10}children:\[(.),/,
```

Reply:
use \i, not .